cpp/unreal_plugins/SpEngine/Source/SpEngine/GameWorldService.h

```diff
@@ -41,6 +61,111 @@ class GameWorldService {
             SP_ASSERT(world_);
             return Unreal::toStdString(world_->GetName());
         });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "find_actors", [this]() -> std::vector<uint64_t> {
```

Comment:
I admit that our current guidelines and existing conventions don't offer clear formatting guidance in the case of these entry points. I think the following formatting is a bit easier to read and skim. Lambda always on a new line for consistency, even though sometimes the signature would be short enough to squeeze onto the previous line. Closing parenthesis on a new line aligned with the capture.

```
        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "set_game_paused",
            [this](const bool& paused) -> void {
                SP_ASSERT(world_);
                UGameplayStatics::SetGamePaused(world_, paused);
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "open_level",
            [this](const std::string& level_name) -> void {
                SP_ASSERT(world_);
                SP_LOG("Opening level: ", level_name);
                UGameplayStatics::OpenLevel(world_, Unreal::toFName(level_name));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_world_name",
            [this]() -> std::string {
                SP_ASSERT(world_);
                return Unreal::toStdString(world_->GetName());
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "find_actors",
            [this]() -> std::vector<uint64_t> {
                return toUint64(Unreal::findActors(world_));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "find_actors_as_map",
            [this]() -> std::map<std::string, uint64_t> {
                return toUint64(Unreal::findActorsAsMap(world_));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_components",
            [this](const uint64_t& actor) -> std::vector<uint64_t> {
                return toUint64(Unreal::getComponents(reinterpret_cast<AActor*>(actor)));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_components_as_map",
            [this](const uint64_t& actor) -> std::map<std::string, uint64_t> {
                return toUint64(Unreal::getComponentsAsMap(reinterpret_cast<AActor*>(actor)));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_children_components",
            [this](const uint64_t& parent, const bool& include_all_descendants) -> std::vector<uint64_t> {
                return toUint64(Unreal::getChildrenComponents(reinterpret_cast<USceneComponent*>(parent), include_all_descendants));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_children_components_as_map",
            [this](const uint64_t& parent, const bool& include_all_descendants) -> std::map<std::string, uint64_t> {
                return toUint64(Unreal::getChildrenComponentsAsMap(reinterpret_cast<USceneComponent*>(parent), include_all_descendants));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_object_properties_as_string_from_object",
            [this](const uint64_t& uobject) -> std::string {
                return Unreal::getObjectPropertiesAsString(reinterpret_cast<UObject*>(uobject));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_object_properties_as_string_from_struct",
            [this](const uint64_t& value_ptr, const uint64_t& ustruct) -> std::string {
                return Unreal::getObjectPropertiesAsString(reinterpret_cast<void*>(value_ptr), reinterpret_cast<UStruct*>(ustruct));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "set_object_properties_from_string_for_object",
            [this](const uint64_t& uobject, const std::string& string) -> void {
                Unreal::setObjectPropertiesFromString(reinterpret_cast<UObject*>(uobject), string);
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "set_object_properties_from_string_for_struct",
            [this](const uint64_t& value_ptr, const uint64_t& ustruct, const std::string& string) -> void {
                Unreal::setObjectPropertiesFromString(reinterpret_cast<void*>(value_ptr), reinterpret_cast<UStruct*>(ustruct), string);
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "find_property_by_name_from_object",
            [this](const uint64_t& uobject, const std::string& name) -> GameWorldServicePropertyDesc {
                return toPropertyDesc(Unreal::findPropertyByName(reinterpret_cast<UObject*>(uobject), name));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "find_property_by_name_from_struct",
            [this](const uint64_t& value_ptr, const uint64_t& ustruct, const std::string& name) -> GameWorldServicePropertyDesc {
                return toPropertyDesc(Unreal::findPropertyByName(reinterpret_cast<void*>(value_ptr), reinterpret_cast<UStruct*>(ustruct), name));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_property_value_as_string",
            [this](const GameWorldServicePropertyDesc& game_world_property_desc) -> std::string {
                return Unreal::getPropertyValueAsString(toPropertyDesc(game_world_property_desc));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "set_property_value_from_string",
            [this](const GameWorldServicePropertyDesc& game_world_property_desc, const std::string& string) -> void {
                Unreal::setPropertyValueFromString(toPropertyDesc(game_world_property_desc), string);
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "find_function_by_name",
            [this](const uint64_t& uclass, const std::string& name, const EIncludeSuperFlag::Type& include_super_flag) -> uint64_t {
                return reinterpret_cast<uint64_t>(Unreal::findFunctionByName(reinterpret_cast<UClass*>(uclass), name, include_super_flag));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "call_function",
            [this](const uint64_t& uobject, const uint64_t& ufunction, const std::map<std::string, std::string>& args) -> std::map<std::string, std::string> {
                return Unreal::callFunction(reinterpret_cast<UObject*>(uobject), reinterpret_cast<UFunction*>(ufunction), args);
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "find_special_struct_by_name",
            [this](const std::string& name) -> uint64_t {
                return reinterpret_cast<uint64_t>(Unreal::findSpecialStructByName(name));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "actor_has_stable_name",
            [this](const uint64_t& actor) -> bool {
                return Unreal::hasStableName(reinterpret_cast<AActor*>(actor));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "component_has_stable_name",
            [this](const uint64_t& component) -> bool {
                return Unreal::hasStableName(reinterpret_cast<UActorComponent*>(component));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_stable_name_for_actor",
            [this](const uint64_t& actor) -> std::string {
                return Unreal::getStableName(reinterpret_cast<AActor*>(actor));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_stable_name_for_actor_component",
            [this](const uint64_t& actor_component, const bool& include_actor_name) -> std::string {
                return Unreal::getStableName(reinterpret_cast<UActorComponent*>(actor_component), include_actor_name);
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_stable_name_for_scene_component",
            [this](const uint64_t& scene_component, const bool& include_actor_name) -> std::string {
                return Unreal::getStableName(reinterpret_cast<USceneComponent*>(scene_component), include_actor_name);
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_actor_tags",
            [this](const uint64_t& actor) -> std::vector<std::string> {
                return Unreal::getTags(reinterpret_cast<AActor*>(actor));
            });

        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_component_tags",
            [this](const uint64_t& component) -> std::vector<std::string> {
                return Unreal::getTags(reinterpret_cast<UActorComponent*>(component));
            });
```

Reply:
This convention looks good to me. I wasn't sure if there was a need for a convention in this case as only few functions had to be formatted into a new line, but looking at this, it feels that this is better as it's uniform. I don't mind following this convention in all cases.